java/src/main/java/com/powsybl/python/SensitivityAnalysisResultContextV1.java

```diff
@@ -43,37 +45,62 @@
 
     @Override
     public PyPowsyblApiHeader.MatrixPointer createSensitivityMatrix(String contingencyId) {
```

Comment:
I encountered an "index out of bound" while testing with a large network and "open load flow" v1 : 
```
_pypowsybl.PyPowsyblError: Index 398412 out of bounds for length 370202
```
I suspect that in some cases, the collection of `SensitivityValue` is smaller than (functions*variables) : in that case, the computed index may be greater than `sensitivityValues.length`.

Reply:
I mean: it's not an issue for open load flow, since in reality we will use v2, but I think we need to handle the cases where some sensitivity values are missing.
Maybe we should fill the result with `NaN` in those cases ? (is it technically possible to pass `NaN` from java to python? )